### PR TITLE
GH60: GH64: Cleanup code.

### LINKED ISSUE #60: Cleanup code

### LINKED ISSUE #64: Add StyleCop

### PR DESCRIPTION
**Pull request by mholo65**
Cleanup code and add StyleCop.Analyzers

- Fixes #60
- Fixes #64

### CODE REVIEW

File: src/Test.ruleset [hunk 1]
```diff
@@ -0,0 +1,49 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="Cake Ruleset" ToolsVersion="12.0">
```
Comment (patriksvensson) [inline src/Test.ruleset]: Perhaps rename to `Cake Bakery Test Ruleset`?

File: src/Cake.ruleset [hunk 2]
```diff
@@ -0,0 +1,36 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="Cake Ruleset" ToolsVersion="12.0">
```
Comment (patriksvensson) [inline src/Cake.ruleset]: Perhaps rename to `Cake Bakery Ruleset`?

File: src/Test.ruleset (hunk 1 above)
Comment (mholo65) [reply src/Test.ruleset]: Don't know. Kind of like that it says Cake, as it's copy-pasted from the Cake repo 😉

**Comment by patriksvensson** [comment]
Merged as promised. Thank you for your contribution 👍 😉

**Pull request merged**